src/AWS.Deploy.CLI/ServerMode/Models/TypeHintResourceColumn.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.CLI.ServerMode.Models
+{
+    /// <summary>
+    /// Represents a column header for a list/grid of <see cref="TypeHintResourceSummary"/> rows
+    /// </summary>
+    public class TypeHintResourceColumn
+    {
+        /// <summary>
+        /// Internal Id for the column. This can be used as a key to access a specific
+        /// row's value via its <see cref="TypeHintResourceSummary.ColumnValues"/>
+        /// </summary>
+        public string Id { get; set; }
```

Comment:
Based on my earlier comment, I think we don't need this class.

Reply:
I wanted to model the columns as classes as opposed to strings. That way if we add additional properties in the future (width, sort order, etc.), it'd be an additive change to an existing class as opposed to a breaking change for consumers of server mode to transition from `List<string>` to `List<SomeNewClass>`.

We could use a simpler model for the CLI-specific classes instead of the `/ServerMode/` ones, but I opted to keep them as similar as possible.